python/lsst/obs/base/ingest_tests.py

```diff
@@ -30,8 +30,8 @@
 import os
 import shutil
 
-from lsst.daf.butler import Butler
-from lsst.obs.base import RawIngestTask
+import lsst.daf.butler
```

Comment:
I'm not really happy with these changes. Butler is an obvious class and doesn't need the clutter of saying `lsst.daf.butler.Butler`. At the very least these style changes should be in a separate commit.